lua/neogit/lib/hl.lua

```diff
@@ -155,6 +155,9 @@ function M.setup()
     NeogitCommandCodeNormal = { link = "String" },
     NeogitCommandCodeError = { link = "Error" },
     NeogitBranch = { fg = palette.orange, bold = true },
+    NeogitHeadOid = { link = "Comment" },
```

Comment:
Can these just link to `NeogitObjectId` instead of `Comment`? I wanna make this easier on theme-authors.